src/main/java/uk/gov/ons/ctp/response/action/service/ActionStateService.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.ons.ctp.response.action.service;
+
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.ons.ctp.response.action.domain.model.Action;
+import uk.gov.ons.ctp.response.action.domain.repository.ActionRepository;
+import uk.gov.ons.ctp.response.action.representation.ActionDTO;
+import uk.gov.ons.ctp.response.lib.common.error.CTPException;
+import uk.gov.ons.ctp.response.lib.common.state.StateTransitionManager;
+import uk.gov.ons.ctp.response.lib.common.time.DateTimeUtil;
+
+@Service
+public class ActionStateService {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ActionStateService.class);
```

Comment:
Lower case `log` here.  It's consistently lowercase throughout the app

Reply:
Its wrong elsewhere in the application and drives me crazy, Java constants identified by "static final" should be upper case. I'm tempted to leave this one

Reply:
thats fine, I thought this was the only place in the code where it's upper.  If it's some upper and some lower, but upper is correct, then I'd just leave this as is.